src/server/graphql.js

```diff
@@ -0,0 +1,70 @@
+import { GraphQLScalarType } from 'graphql';
+import { Kind } from 'graphql/language';
+import schemas from '../schema';
+
+const resolverMap = {
+  Query: {
+    getDecisions: () => {
+      const doc = schemas.Decision.find();
+      return doc;
+    },
+
+    countDecisions: () => schemas.Decision.where({}).count(),
+
+    countCCRR: (_, { table }) => {
+      if (table) {
+        return schemas.CrossCuttingResearchRow.where({ table }).count();
+      }
+      return schemas.CrossCuttingResearchRow.where({}).count();
+    },
+
+    countMandates: () => schemas.Mandate.where({}).count(),
+  },
+
+  Mutation: {
+    createDecision: (_, { decision }) => {
+      const doc = new schemas.Decision(decision);
+      if (doc.validateSync()) {
+        // TODO
+      }
+      doc.save();
+      return doc;
+    },
+
+    createCCRR: (_, { row }) => {
+      const doc = new schemas.CrossCuttingResearchRow(row);
+      // TODO: use validation to check if we can insert it
+      doc.save();
+      return doc;
+    },
+
+    createMandate: (_, { mandate }) => {
+      const doc = new schemas.Mandate(mandate);
+      const errors = doc.validateSync();
+      if (errors) {
+        console.log(errors);
+        // TODO
+      }
+      doc.save();
```

Comment:
Because I'm not sure the best approach w.r.t how GraphQL / Node handle errors, so I don't know what error handling will give us the best error messages. Suggestions? This will be handled in a new PR.